moveit_ros/hybrid_planning/global_planner/global_planner_plugins/include/moveit/global_planner/moveit_planning_pipeline.h

```diff
@@ -50,7 +50,7 @@ class MoveItPlanningPipeline : public GlobalPlannerInterface
 {
 public:
   MoveItPlanningPipeline(){};
```

Comment:
Why not using this:

```suggestion
  MoveItPlanningPipeline() = default;
```

Anyway, regardless of the style semi-colon is redundant:
```suggestion
  MoveItPlanningPipeline(){}
```